tests/app/test_message.py

```diff
@@ -273,6 +274,15 @@ def test_msg_to_validation_invalid_respondent(self):
 
         self.assertTrue(errors == {'msg_to': ['NotAValidUser is not a valid respondent.']})
 
+    def test_message_url_is_created_correctly(self):
+        self.threadId = 'threadid'
+        with self.app.app_context():
+            message_send = MessageSend()
+            response = message_send._create_message_url(self.threadId)
+
+        self.assertTrue(response == {'MESSAGE_URL': 'http://localhost:8082/secure-message/thread/'
```

Comment:
Minor point - may want to make the string the same f string as in the code . Then if values change the test still passes, but if the format of the url changes then the test fails.